src/commands/client/info/index.ts

```diff
@@ -1,17 +1,30 @@
 import { Command } from "@commands/command";
 
 type NamespaceTitle = string;
+type SimilarityFunction = "COSINE" | "EUCLIDEAN" | "DOT_PRODUCT";
 type NamespaceInfo = {
   vectorCount: number;
   pendingVectorCount: number;
 };
 
+type DenseIndexInfo = {
+  dimension: number;
+  similarityFunction: SimilarityFunction;
+  embeddingModel: string;
+}
+
+type SparseIndexInfo = {
+  embeddingModel: string;
```

Comment:
aren't the embeddingModel fields optional? They should be when the user chooses custom I think